marketpulse/devices/models.py

```diff
@@ -5,10 +5,12 @@ class Device(models.Model):
     """Model for FfxOS devices data."""
 
     model = models.CharField(max_length=120)
-    manufacturer = models.CharField(max_length=120)
+    manufacturer = models.CharField(max_length=120, blank=True, null=True)
```

Comment:
Why did you add here both `blank` and `null`. Shouldn't a device always have a manufacturer either is a FxOS device or not?